cmd/veto/models.go

```
@@ -54,6 +54,18 @@ func localModelsPath() string {
 	return filepath.Join(home, ".veto", "models.json")
 }
 
+func openRouterCatalogCachePath() string {
```

golangci-lint (2.12.2)

[error] func openRouterCatalogCachePath is unused

(unused)